index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <link rel="stylesheet" href="css/style.css" />
+        <title>Apple</title>
+        <meta charset="utf-8" />
+    </head>
+    <body>
+        <main>
+                    <header class="header">
+                            <div id="menu-outer">
+                            <div class="table">
+                                <ul id="horizontal-list">
+                                    <li><a href="#"><div></div></a></li>
+                                    <li><a href="#">Store</a></li>
+                                    <li><a href="#">Mac</a></li>
+                                    <li><a href="#">Ipod</a></li>
+                                    <li><a href="#">Iphone</a></li>
+                                    <li><a href="#">Iped</a></li>
+                                    <li><a href="#">Itunes</a></li>
+                                    <li><a href="#">Support</a></li>
+                                    <li><input type="text"></li>
+                                </ul>
+                            </div>
+                        </div>
+                        
+                    </header>
+                    <div class="text">
+                        <img class="text1" src="img/text1.png" width="980" height="100" alt="What will your verse be?">
+                        <img class="text2" src="img/text2.png" width="980" height="82" alt="Everyone has something to share. See how people are using iPad in their unique ways to add to the world’s story.">
+                        <img class="text3" src="img/text3-b.png" width="980" height="32" alt="Watch the “Your Verse” film and explore the stories">
+                    </div>
+        </main>
+        <div class="contentN">
+            <div class="conUl">
+                <a href="#"><div></div></a>
```

Comment:
While this the proper markup setup I.E: `<div>` tags inside an `<a>` tag, the background images should be set to the `<div>` tag and not the `<a>` elements. Kindly make that small adjustment.